resources/wiki-overrides.properties

```diff
@@ -4,9 +4,6 @@
 # Required until a newer version than 1.1 is released
 archived-artifact-url-viewer=https://wiki.jenkins-ci.org/display/JENKINS/Archived+Artifact+Url+Viewer+PlugIn
 
-# Required until a newer version than 0.3 is released
-plugin-usage-plugin=https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Usage+Plugin+(Community)
```

Comment:
https://github.com/jenkinsci/plugin-usage-plugin/commit/c135ff6c5ca1d4f46bd1c6f576a70c9059bdc2b3 references the repo now. Changelog was migrated to a different file, the rest is identical.